x-pack/plugins/security_solution/public/management/pages/policy/view/ingest_manager_integration/endpoint_package_custom_extension/components/fleet_trusted_apps_card.test.tsx

```diff
@@ -0,0 +1,117 @@
+import { I18nProvider } from '@kbn/i18n/react';
+import { FleetTrustedAppsCard } from './fleet_trusted_apps_card';
+import * as reactTestingLibrary from '@testing-library/react';
+import { TrustedAppsHttpService } from '../../../../../trusted_apps/service';
+import { useToasts } from '../../../../../../../common/lib/kibana';
+import { getMockTheme } from '../../../../../../../../public/common/lib/kibana/kibana_react.mock';
+import { GetExceptionSummaryResponse } from '../../../../../../../../common/endpoint/types';
+
+jest.mock('./exception_items_summary');
+jest.mock('../../../../../trusted_apps/service');
+
+jest.mock('../../../../../../../../../../../src/plugins/kibana_react/public', () => {
+  const originalModule = jest.requireActual(
+    '../../../../../../../../../../../src/plugins/kibana_react/public'
+  );
+  const useKibana = jest.fn().mockImplementation(() => ({
+    services: {
+      http: {},
+      data: {},
+      notifications: {},
+      application: {
+        getUrlForApp: jest.fn(),
+      },
+    },
+  }));
+
+  return {
+    ...originalModule,
+    useKibana,
+  };
+});
+
+jest.mock('../../../../../../../common/lib/kibana');
+
+const mockTheme = getMockTheme({
+  eui: {
+    paddingSizes: { m: '2' },
+  },
+});
+
+const TrustedAppsHttpServiceMock = TrustedAppsHttpService as jest.Mock;
+const useToastsMock = useToasts as jest.Mock;
+
+const summary: GetExceptionSummaryResponse = {
+  windows: 3,
+  linux: 2,
+  macos: 2,
+  total: 7,
+};
+
+describe('Fleet trusted apps card', () => {
+  let promise: Promise<GetExceptionSummaryResponse>;
+  let addDanger: jest.Mock = jest.fn();
+  const renderComponent: () => Promise<reactTestingLibrary.RenderResult> = async () => {
+    const Wrapper: React.FC = ({ children }) => (
+      <I18nProvider>
+        <ThemeProvider theme={mockTheme}>{children}</ThemeProvider>
+      </I18nProvider>
+    );
+    // @ts-ignore
+    const component = reactTestingLibrary.render(<FleetTrustedAppsCard />, { wrapper: Wrapper });
+    try {
+      // @ts-ignore
+      await reactTestingLibrary.act(() => promise);
+    } catch (err) {
+      return component;
+    }
+    return component;
+  };
+
+  afterEach(() => {
+    TrustedAppsHttpServiceMock.mockReset();
+  });
+  beforeEach(() => {
+    promise = Promise.resolve(summary);
+    addDanger = jest.fn();
+  });
+  beforeAll(() => {
+    useToastsMock.mockImplementation(() => {
+      return {
+        addDanger,
+      };
+    });
+  });
+  it('should renders correctly', async () => {
+    TrustedAppsHttpServiceMock.mockImplementationOnce(() => {
+      return {
+        getTrustedAppsSummary: () => jest.fn(() => promise),
+      };
+    });
+    const component = await renderComponent();
+    expect(component.getByText('Trusted Applications')).not.toBeNull();
+    expect(component.getByText('Manage trusted applications')).not.toBeNull();
+  });
+  it('should renders an error when api call fails', async () => {
```

Comment:
nitpick: I think it's okay to have test descriptions with even more context. This one renders an error toast and should say something like `should render an error toast when...`. That way one doesn't have to read the code to know what this test is verifying.